docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

```diff
@@ -11,13 +11,123 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-7.14.1>>
+
 * <<release-notes-7.14.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 7.14.1 relnotes
+
+[[release-notes-7.14.1]]
+== {fleet} and {agent} 7.14.1
+
+Review important information about the {fleet} and {agent} 7.14.1 releases.
+
+//[discrete]
+//[[security-updates-7.14.1]]
+//=== Security updates
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+//[discrete]
+//[[breaking-changes-7.14.1]]
+//=== Breaking changes
+
+//Breaking changes can prevent your application from optimal operation and
+//performance. Before you upgrade, review the breaking changes, then mitigate the
+//impact to your application.
+
+//[discrete]
+//[[breaking-PR#]]
+//.Short description
+//[%collapsible]
+//====
+//*Details* +
+//<Describe new behavior.> For more information, refer to {kibana-pull}PR[#PR].
+
+//*Impact* +
+//<Describe how users should mitigate the change.> For more information, refer to {fleet-guide}/fleet-server.html[Fleet Server].
+//====
+
+//[discrete]
+//[[known-issues-7.14.1]]
+//=== Known issues
+
+//[[known-issue-issue#]]
+//.Short description
+//[%collapsible]
+//====
+
+//*Details* 
+
+//<Describe known issue.>
+
+//*Impact* +
+
+//<Describe impact or workaround.>
+
+//====
+
+//[discrete]
+//[[deprecations-7.14.1]]
+//=== Deprecations
+
+//The following functionality is deprecated in 7.14.1, and will be removed in
+//8.0.0. Deprecated functionality does not have an immediate impact on your
+//application, but we strongly recommend you make the necessary updates after you
+//upgrade to 7.14.1.
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+//[discrete]
+//[[new-features-7.14.1]]
+//=== New features
+
+//The 7.14.1 release adds the following new and notable features.
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+//[discrete]
+//[[enhancements-7.14.1]]
+//=== Enhancements
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+[discrete]
+[[bug-fixes-7.14.1]]
+=== Bug fixes
+
+{fleet}::
+* Fixes integrations count in category facet {kibana-pull}107652[#107652]
+
+//{agent}::
+//* add info
```

Comment:
* Fix policies not being assigned to agents (flaky)
https://github.com/elastic/beats/pull/27362
https://github.com/elastic/beats/pull/27297

in this case there are multiple PRs related to this fix, not sure what we do for formatting here. @ruflin are there any other important 7.14.1 fixes?

Reply:
We can just add multiple links at the end. I will add this item. Thanks!